### PR TITLE
chore(docs): Update release notes to document change to CLI and kubeconfig file required changes

### PR DESCRIPTION
**Pull request by jasonmcintosh**

### CODE REVIEW

File: content/en/docs/releases/next-release-preview/index.md [hunk 1]
```diff
@@ -137,3 +137,15 @@ resilience4j.circuitbreaker:
...
       waitDurationInOpenState: 9000
 ```
+
+### Clouddriver
+There are several minor updates to the CLIs in use by clouddriver for both slim and ubuntu images
+* Kubectl to 1.22
+* AWS CLI to 1.22
+
+### FUTURE BREAKING CHANGE:
+kubectl in the latest releases have removed support for token aquisition using the `apiVersion: client.authentication.k8s.io/v1alpha1` exec API.  The aws cli version 1.22 does not support this new beta1 syntax yet but the aws-iam-authenticator supports both v1alpha1 and v1beta1 exec APIs.  It's recommended that
```
Comment (dbyron-sf) [inline content/en/docs/releases/next-release-preview/index.md:147]: Isn’t there something more specific we can say?  If version 1.22 of the aws cli doesn't work with k8s >= 1.24, then this isn't a future breaking change, it's a present breaking change, right?

As I understand it, it’s important to have a “new enough" aws-iam-authenticator to target k8s clusters >= 1.24, where "new enough" means no longer references `client.authentication.k8s.io/v1alpha1 ExecCredential`.  https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md even says:
> The client.authentication.k8s.io/v1alpha1 ExecCredential has been removed.

Clouddriver currently brings in version [0.5.9 of aws-iam-authenticator in slim](https://github.com/spinnaker/clouddriver/blob/dbc3dcbd1459be4eebdfcb1aa6b0cb075dd4734d/Dockerfile.slim#L9) and [1.16.8? in ubuntu](https://github.com/spinnaker/clouddriver/blob/dbc3dcbd1459be4eebdfcb1aa6b0cb075dd4734d/Dockerfile.ubuntu#L28).  I'm not sure where the corresponding source is for 1.16.8, but 0.5.9 is already new enough.  See [here](https://git.soma.salesforce.com/spinnaker/aws-iam-authenticator/blob/v0.5.9/pkg/token/token.go#L344) that already references v1beta1.

Is there a consequence of using version 1.22 of the aws cli with older eks clusters? From what I can tell, `client.authentication.k8s.io/v1beta1` of ExecCredential first appeared in [this commit](https://github.com/kubernetes/kubernetes/commit/8952b8f7c99b75a9dc1aa47eb50770d218b2a8bf), which was first released (according to git tags) in k8s 1.11.0 although the [1.11 changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.11.md) doesn't mention it.

Comment (dbyron-sf) [reply content/en/docs/releases/next-release-preview/index.md:147]: Let's remove the aws cli then?

Comment (jasonmcintosh) [reply content/en/docs/releases/next-release-preview/index.md:147]: Sadly it IS still used :( the aws-iam-authenticator is more performant, and I'd ABSOLUTELY prefer that.  BUT the other place it's used is for fetching ECR tokens for docker containers.  https://docs.armory.io/continuous-deployment/armory-admin/aws/artifacts-ecr-connect/ e.g. this.

Comment (dbyron-sf) [reply content/en/docs/releases/next-release-preview/index.md:147]: Hmmm...it certainly is nice to have the aws cli in clouddriver for troubleshooting all manner of aws issues.  I'd really like the text here to be a bit more crisp though.  Is there something definitive we can say for 1.32?  Did we break something/change behavior?  If there's something people need to do when we release 1.33, I'd say let's wait for 1.33 so we can make sure we get it right.  Predicting the future makes me nervous.

Comment (jasonmcintosh) [reply content/en/docs/releases/next-release-preview/index.md:147]: Nothing's broken yet ... was just "hey we should start warning people" :) Happy to change the text to something else... RIGHT NOW, nothing has changed.  just hoping to bump later.

Comment (jasonmcintosh) [reply content/en/docs/releases/next-release-preview/index.md:147]: Leave this for the next release then and update the text to be more ... direct?

**Comment by jasonmcintosh** [comment]
Closing - have a new PR with this.

**Pull request closed without merging**